### PR TITLE
Removes xwt support from WB update site

### PR DESCRIPTION
**Pull request by vogella**
We are planning to remove the unmaintained and broken XWT support. As a first step, lets remove it from the update site.

### CODE REVIEW

**Comment by vogella** [comment]
First step for #362

File: org.eclipse.wb.releng.updatesite/category.xml [hunk 1]
```diff
@@ -36,17 +33,9 @@
    <feature id="org.eclipse.wb.swt.feature" version="0.0.0">
       <category name="org.eclipse.wb"/>
    </feature>
-   <feature id="org.eclipse.wb.xwt.feature" version="0.0.0">
-      <category name="org.eclipse.wb.xwt"/>
-   </feature>
    <category-def name="org.eclipse.wb" label="WindowBuilder">
       <description>
          This is all of WindowBuilder. You can safely install all features of this section. If you are looking for XWT support, uncheck &quot;Group by category&quot;.
```
Comment (ptziegler) [inline org.eclipse.wb.releng.updatesite/category.xml:38]: I think the following part can be removed from the description as well:
> If you are looking for XWT support, uncheck &quot;Group by category&quot;.

**Comment by vogella** [comment]
Thanks @ptziegler text is removed now.

**Review by ptziegler — APPROVED** [review]
LGTM

**Pull request merged**